src/components/Backgrounds/SeriaBackground.tsx

```diff
@@ -0,0 +1,16 @@
+import React from "react";
+import Slider from "../Slider";
+
+const SeriaBackground = () => {
+    return(
+        <div className="w-full h-screen bg-center bg-cover bg-no-repeat"
+            style={{ backgroundImage: 'url("/assets/seriabg.jpg")' }}>
+            <div className="w-full h-full bg-bottom bg-no-repeat"
+                style={{ backgroundImage: 'url("/assets/seria1.png")', backgroundSize: '42%', backgroundPosition: '42% bottom'}}>
+                <Slider Color="#FFD700" Colors="#B8860B" Colorl="#F4A460"/>
+            </div>
+        </div>
+    )
+}
+
+export default SeriaBackground;
```

Comment:
```suggestion
import React from "react";
import Slider from "../Slider";

const SeriaBackground = () => {
    return(
        <div className="w-full h-screen bg-center bg-cover bg-no-repeat"
            style={{ backgroundImage: 'url("/assets/seriabg.jpg")' }}>
            <div className="w-full h-full bg-bottom bg-no-repeat"
                style={{ backgroundImage: 'url("/assets/seria1.png")', backgroundSize: '42%', backgroundPosition: '42% bottom'}}>
                <Slider Color="#FFD700" Colors="#B8860B" Colorl="#F4A460"/>
            </div>
        </div>
    )
}

export default SeriaBackground;
```

De cette manière, tu pourrais faire un composant générique background où tu pourrais juste envoyer en props toute les infos dont tu as besoin au lieu de refaire un nouveau fichier à chaque fois :+1:

Reply:
j'ai pas compris ce que tu as modifier mais oui comme dit plus haut c'est le but

Reply:
Ah ca a pas fait mes modifs, je vais edit

Reply:
```suggestion
import React from "react";
import Slider from "../Slider";

const SeriaBackground = () => {
    return(
        <div className="w-full h-screen bg-center bg-cover bg-no-repeat"
            style={{ backgroundImage: 'url("/assets/seriabg.jpg")' }}>
            <div className="w-full h-full bg-bottom bg-no-repeat"
                style={{ backgroundImage: 'url("/assets/seria1.png")', backgroundSize: '42%', backgroundPosition: '42% bottom'}}>
                <Slider Color="#FFD700" Colors="#B8860B" Colorl="#F4A460"/>
            </div>
        </div>
    )
}

export default SeriaBackground;
```

Reply:
ah le userSelect, qui switch à chaque refresh ouais why not

Reply:
Et surtout, plus besoin des histoires de 42%, tout est bien centré de base